package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "armlet",
-  "version": "1.2.2-git",
+  "version": "2.0.0",
```

Comment:
Le't use 2.0.0-git or something like this. Before the final release we'll drop the `-git`.

Reply:
Done